src/main/java/nextstep/courses/domain/Capacity.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.courses.domain;
+
+import static nextstep.util.NullValidator.validateNull;
+
+public class Capacity {
+    private final Positive capacity;
+
+    public Capacity(int capacity) {
+        this(new Positive(capacity));
+    }
+
+    public Capacity(Positive capacity) {
+        validateNull(capacity);
+        this.capacity = capacity;
+    }
+
+    public boolean canEnroll(int size) {
+        return capacity.isGraterThan(new Positive(size));
```

Comment:
```suggestion
        return capacity.isGraterThan(new Positive(size));
        return capacity.isGraterThan(size);
```
Positive 객체에 두 개의 타입을 가지는 isGraterThan() 메서드를 가지면 이 메서드를 사용하는 개발자 입장에서 타입 변환을 하지 않아도 되기 때문에 좋지 않을까?